### ISSUE TITLE
Scoping issue with response transformations

### ISSUE THREAD

**Issue by mitchelloharawild** [tidyverts/fabletools#207]
``` r
library(fpp3)
#> ── Attaching packages ──────────────────────────────────────────────────────────────────────── fpp3 0.2 ──
#> ✓ tibble      3.0.1     ✓ tsibble     0.8.6
#> ✓ dplyr       0.8.5     ✓ tsibbledata 0.1.0
#> ✓ tidyr       1.0.2     ✓ feasts      0.1.3
#> ✓ lubridate   1.7.8     ✓ fable       0.2.0
#> ✓ ggplot2     3.3.0
#> ── Conflicts ─────────────────────────────────────────────────────────────────────────── fpp3_conflicts ──
#> x lubridate::date()   masks base::date()
#> x dplyr::filter()     masks stats::filter()
#> x tsibble::id()       masks dplyr::id()
#> x tsibble::interval() masks lubridate::interval()
#> x dplyr::lag()        masks stats::lag()
uad <- as_tsibble(USAccDeaths)
lambda <- uad %>% features(value, guerrero)
fit <- uad %>% 
  model(ARIMA(box_cox(value, lambda$lambda_guerrero)))
forecast(fit)
#> Error in is_null(msg): object 'error' not found
```

<sup>Created on 2020-05-28 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by mitchelloharawild** [comment]
This is a long running issue, due to `all.vars()` thinking `lambda_guerrero` needs to exist somewhere not in `lambda`.

**Comment by mitchelloharawild** [comment]
Fixed by 395b657955456a7be6f13713e29952ba0a1fa872
